owncloud/version.ini

```diff
@@ -0,0 +1,5 @@
+[General]
+name = ownCloud
+branches = master;2.6
+tarballs = 2.5.4.11654
```

Comment:
What does this mean?

Reply:
Thats the last stable tarball release we had and yes our versioning seems to be special ❄️